src/components/input/index.tsx

```diff
@@ -99,13 +104,39 @@ const Input = forwardRef<HTMLInputElement, Props>(
       visible: !!value,
       disabled,
       onClear: () => {
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        ;(inputRef.current as any).value = ""
         onChange({ target: { name, value: "", cleared: true } })
       },
     }
 
+    const debouncedChangeHandler = useMemo(() => {
```

Comment:
what is the `useMemo` doing in this case?

Reply:
Well it was between `useMemo` and `useCallback` because we need to make sure the `debounce` instance is the same between re-renders. 
`useMemo` also calls debounce() only during initial rendering of the component, so yea, it provides a bit better performance.